tests/unit/components/search-autocomplete/SearchAutocomplete.spec.jsx

```diff
@@ -17,41 +16,78 @@ describe('SearchAutocomplete', () => {
         textFieldProps={{ label: 'Search' }}
       />
     )
-  })
 
-  it('renders autocomplete with search input', () => {
     const searchInput = screen.getByLabelText('Search')
     expect(searchInput).toBeInTheDocument()
   })
 
+  it('renders autocomplete with search input value', () => {
```

Comment:
It seems like you have previous version of this file, it shouldn`t have renders in each test, try to fix it

Reply:
i return old test, now its passing, before somehow not.